x-pack/plugins/alerting/server/routes/rule/apis/mute_alert/mute_alert.ts

```diff
@@ -26,7 +26,9 @@ export const muteAlertRoute = (
         summary: `Mute an alert`,
       },
       validate: {
-        params: muteAlertParamsSchemaV1,
+        request: {
+          params: muteAlertParamsSchemaV1,
```

Comment:
Should we specify the `403` here? Since we have the `RuleTypeDisabledError` in 42, which sends out a custom error body. If so we should version that schema well

Reply:
@lcawl what do you think?

Reply:
@cnasikas has created an [issue](https://github.com/elastic/kibana/issues/188514) to handle 4xx and 5xx errors for all routes. 
Is it fine if I handle error response and its schema versioning in a separate PR?

Reply:
Yes, that seems fine to me!